packages/utils/src/bulk-insert-patients.ts

```diff
@@ -104,7 +104,8 @@ function toTitleCase(str: string): string {
     .trim();
 }
 
-function normalizeGender(gender: string): "M" | "F" {
+function normalizeGender(gender: string | undefined): "M" | "F" {
+  if (gender == undefined) throw new Error(`Missing gender`);
```

Comment:
nit: === ?

Reply:
Intentionally `==` so it catches both `undefined` and `null`